FreydCategoriesForCAP/gap/CategoryOfRows.gi

```diff
@@ -54,6 +54,8 @@ InstallMethod( CategoryOfRows,
     SetIsAdditiveCategory( category, true );
     
     SetIsRigidSymmetricClosedMonoidalCategory( category, true );
+
```

Comment:
Two tiny remarks:

1) Rows( R ) should only be monoidal w.r.t. the given operations if R is commutative.

2) I'm not sure if the names are my fault. But every rigid symmetric category is always both closed and coclosed.

Reply:
1. Should be fixed now.
2. It seems nobody thought about that. For now, I guess it's easier to leave it as it is, and I will add that to the meeting with @mohamed-barakat next Thursday.

Reply:
I created an Issue for 2.: #803